oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -166,6 +165,24 @@ public static GoogleCredentials fromStream(InputStream credentialsStream,
   }
 
   /**
+   * Saves the end user credentials in specified file
```

Comment:
Why do you want this?  Didn't you get them from a file or some other mechanism?

What kind of credentials are you trying to save, many are short lived.

Reply:
The main purpose is to store refresh tokens as described in this issue: https://github.com/google/google-auth-library-java/issues/54